fedora/main.fmf

```diff
@@ -0,0 +1,26 @@
+# CI configuration to run upstream tests in Fedora ecosystem
+# More info:
+# Testing framework: http://tmt.readthedocs.io/
+# CI system docs: https://packit.dev/testing-farm/
+
+summary: Run tests on Fedora Linux
+
+discover:
+    how: shell
+    tests:
+      - name: Upstream test suite
+        test: make test
+
+provision:
+    how: local
+
+prepare:
+  - name: packages
+    how: install
+    package:
+      - python3-pytest
+      - python3-twisted
+      - make
+
+execute:
+    how: tmt
```

Comment:
```suggestion
```
`tmt`  is default, there should be no need to write it. Therefore it can be left blank.

But the `execute:` line is mandatory for all plans AFAIR.

Reply:
I can drop that (and will hope it won't be confusing to leave just `execute: {}`

Reply:
Without the {} is fine.

(I don't think it's a standard YAML.)

Reply:
Feel free to leave it as it is to avoid confusion, though :).

Reply:
I would vote for keeping `how: tmt` to make it less confusing (and more explicit) 'cause the empty `execute` looks a bit weird. Just my two cents...